index.html

```diff
@@ -1,12 +1,270 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Девайс</title>
-  </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>HTML Academy: Девайс</title>
+</head>
+
+<body>
+
+<header class="site-header">
+
+  <nav class="user-navigation">
+    <img class="logo" src="img\logo.svg" alt="Логотип Device">
```

Comment:
```suggestion
    <img class="logo" src="img/logo.svg" alt="Логотип Device.">
```

пути неправильные у картинок - надо везде поправить, и в альтах надо ставить точки на конце для интонационной паузы для скрин-ридеров - тоже поправь везде